src/main/webapp/feed.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Message Feed</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="/js/public-feed.js"></script>
+  </head>
+  <body onload="buildUI()">
+    <div id="content">
+      <h1>Message Feed</h1>
+      <hr/>
+      <div id="message-container">Loading...</div>
+    </div>
+  </body>
+</html>
```

Comment:
Please add an empty line at the end of the file. It is good practice.

Reply:
ok thanks!